definitions/npm/@testing-library/react_v10.x.x/flow_v0.104.x-/react_v10.x.x.js

```diff
@@ -134,7 +134,7 @@ declare module '@testing-library/react' {
   ) => Promise<HTMLElement>;
 
   declare type FindAllByRole = (
-    role: Matcher,
+    role: any,
```

Comment:
I don't get this.

Reply:
Definitely agree, should be improved, but as a first pass fixing libraries that blocked people from upgrading to latest flow version I would say is more important.

Basically flow already has a massive learning curve, needing most people to maintain lib defs because the repo that holds them are broken isn't a good thing to have in the community, I would expect that if something broke between versions I should be able to run `flow-typed install` immediately to fix it.